configure.ac

```diff
@@ -1687,6 +1687,71 @@ then
 fi
 
 
+# libkcapi
+AC_ARG_ENABLE([kcapi],
```

Comment:
Does it make sense to have `--enable-kcapi` and all the variations of `--enable-kcapi-hmac`, etc without a macro for KCAPI in general? It looks like this enables the AES and HASH , which HMAC, RSA, DH and ECC have separate enables. Is there a better way to do this?